### ISSUE TITLE
Open tiles in new tabs

### ISSUE THREAD

**Issue by siomekziomek** [BlakkM9/speed-dial#2]
Good add-on, great sense of aestethics.

The only thing it really misses is the option for opening tiles in new tabs / in new tabs in background. It's a bit tirying when you have to open new tab over and over again.

Option to set SpeedDial tab name also would be a good idea.

Please think about it :)
Thanks

**Comment by BlakkM9** [comment]
Hey,

Thanks for the feedback! By clicking the tiles with the middle mouse button
you can open the page in a new tab (and depending on your browser settings
this will happen in background or foreground). I just tested this to prove
that it is still working. In my case the scroll icon showed up at the same
time (which should not be the case) but this will be fixed in the next
update if that's the case for you aswell. Maybe thats due to my broken
mouse button but I can't confirm that.

Currently there is a complete rebuild of the addon going on so the next
update will take a while, just so you know.

Regards,
Julian

**Comment by siomekziomek** [comment]
Yeah, I'm aware you can use middle button but I simply can't get used to it no matter what - it's uncomfortable for me no matter what mouse I'm using, not to mention slower. Just a personal preference.

Good luck with rebuild :) Take your time.

Cheers

**Comment by BlakkM9** [comment]
Ok I see. I'll add this as option in the speeddials custom context menu in the rebuild. Something like this was already planned for the implementation of folders (context menu entry to open all pages in that folder).

Thanks! Sorry that you'll need to wait a bit until this feature is in! But I really want to put the code in a better structure because the code got messy over time in the current version. If you're a bit into coding: the rebuild will be based on the angular framework.

Regards

**Comment by ibeladi-mrsool** [comment]
Any updates on this issue?
